index.html

```diff
@@ -79,6 +79,191 @@ <h2>LET'S CONNECT</h2>
+          <img src="images/pic1.png" alt="Tonic project seakpeak">
+        </div>
+        <div class="card-text">
+          <h2>Tonic</h2>
+          <ul class="categories">
+            <li class="canopy">CANOPY</li>
+            <li>•</li>
+            <li>Back End Dev</li>
+            <li>•</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required</p>
+          <ul class="keyword">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="cta-button">See Project</div>
+        </div>
+      </div>
+
+      <div class="card" id="card2">
+        <div class="card-img">
+          <img src="images/pic2.png" alt="MPS project sneakpeak">
+        </div>
+        <div class="card-text">
+          <h2>Multi-Post Stories</h2>
+          <ul class="categories">
+            <li class="canopy">Facebook</li>
+            <li>•</li>
+            <li>Full Stack Dev</li>
+            <li>•</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required</p>
+          <ul class="keyword">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="cta-button">See Project</div>
+        </div>
+      </div>
+      
+      <div class="card" id="card3">
+        <div class="card-img">
+          <img src="images/pic3.png" alt="Facebook 360 sneakpeak">
+        </div>
+        <div class="card-text">
+          <h2>Facebook 360</h2>
+          <ul class="categories">
+            <li class="canopy">Facebook</li>
+            <li>•</li>
+            <li>Full Stack Dev</li>
+            <li>•</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required</p>
+          <ul class="keyword">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="cta-button">See Project</div>
+        </div>
+      </div>
+
+      <div class="card" id="card4">
+        <div class="card-img">
+          <img src="images/pic4.png" alt="Uber navigation sneakpeak">
+        </div>
+        <div class="card-text">
+          <h2>Uber Navigation</h2>
+          <ul class="categories">
+            <li class="canopy">Uber</li>
+            <li>•</li>
+            <li>Lead developer</li>
+            <li>•</li>
+            <li>2018</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required</p>
+          <ul class="keyword">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="cta-button">See Project</div>
+        </div>
```

Comment:
Kindly change this div to a button or a link (a tag). Keep coding 💪

Reply:
done